Polyhedron/demo/Polyhedron/Scene.cpp

```diff
@@ -1002,7 +1002,23 @@ float Scene::get_bbox_length() const
     return bbox().height();
 }
 
+void Scene::printPrimitiveId(QPoint point, CGAL::Three::Viewer_interface* viewer)
+{
+    //TO DO :: Add a if condition to disable this when the printId mode is off
```

Comment:
No, it has been done, bu the branch does not work under Windows, for some
reason, so don't merge it yet.

2015-12-17 18:01 GMT+01:00 Sebastien Loriot <notifications@github.com>:

> In Polyhedron/demo/Polyhedron/Scene.cpp
> <https://github.com/CGAL/cgal/pull/558#discussion_r47930934>:
>
> > @@ -1002,7 +1002,23 @@ float Scene::get_bbox_length() const
> >      return bbox().height();
> >  }
> >
> > +void Scene::printPrimitiveId(QPoint point, CGAL::Three::Viewer_interface* viewer)
> > +{
> > +    //TO DO :: Add a if condition to disable this when the printId mode is off
>
> Is this todo still valid?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CGAL/cgal/pull/558/files#r47930934>.
>